samples/todo-app/deployment.yaml

```diff
@@ -174,3 +170,34 @@ spec:
   ports:
   - port: 80
     targetPort: 80
+---
```

Comment:
Hmm, I don't see the linux nodeselectors that I added a few days back, can you pull the latest version of this file? And consider adding a nodeselector to your deployment?

Reply:
Thanks, I just merged my PR with your change. Should be fixed now. :)